packages/pangea-sdk/pangea/services/intel.py

```diff
@@ -46,7 +47,7 @@ class FileLookupResult(PangeaResponseResult):
     raw_data: Optional[Dict] = None
 
 
-class IPLookupRequest(APIRequestModel):
```

Comment:
in other instances you are keeping the old data types around, why not do that here?

Reply:
Python wouldn't notice it. Other instances are compiled and if we change return type would crash.
Also, in Python, this class is used just internally. In other instances it's exposed to the user